src/inference/Inference.jl

```diff
@@ -61,8 +62,12 @@ export  InferenceAlgorithm,
         dot_observe,
         resume,
         predict,
-        isgibbscomponent
-
+        isgibbscomponent,
+        get_densities_and_θs,     # Tempering 
+        make_tempered_model,
+        make_tempered_logπ,
+        get_θ
```

Comment:
Same here, is this needed?

Reply:
It should be yes, or MCMCTempering's functions would be unable to call them no?

Reply:
I don't understand, can you explain why or how MCMCTempering would call these methods?

Reply:
Yep you're right, haven't worked much with abstract functions before in Julia and had just misunderstood, will delete these from Inference and Turing